plugins/k8sauditeks/README.md

```diff
@@ -0,0 +1,79 @@
+# Kubernetes Audit Events Plugin for EKS
+
+## Introduction
+
+This plugin extends Falco to support [Kubernetes Audit Events](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-backends) from AWS EKS clusters as a new data source. 
+For more details about what Audit logs are, see the [README of k8saudit plugin](https://github.com/falcosecurity/plugins/blob/master/plugins/k8saudit/README.md).
+
+### Functionality
+
+This plugin supports consuming Kubernetes Audit Events stored in Cloudwatch Logs for the EKS Clusters, see [AWS official documentation](https://docs.aws.amazon.com/eks/latest/userguide/control-plane-logs.html) for details.
+
+## Capabilities
+
+The `k8sauditeks` uses the field extraction methods of the [`k8saudit`](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit) plugin as the format for the Audit Logs is same.
+
+### Event Source
+
+The event source for Kubernetes Audit Events from EKS is `k8s_audit`, it allows to use same rule than `k8saudit` plugin.
+
+### Supported Fields
+
+See the list [here](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit#supported-fields).
+
+## Usage
+
+### Configuration
+
+Here's an example of configuration of `falco.yaml`:
+
+```yaml
+plugins:
+  - name: k8sauditeks
+    library_path: libk8sauditeks.so
+    init_config:
+      region: "us-east-1"
+      profile: "default"
+      cluster_name: "my-cluster"
+      shift: 10
+      polling_interval: 10
+      use_async: false
```

Comment:
Let's put this as true by default. It's a huge performance gain. See: https://github.com/falcosecurity/plugin-sdk-go/issues/62

Reply:
ok